sm/plugins/tedge_apt_plugin/src/main.rs

```diff
@@ -107,6 +118,41 @@ fn run_cmd(cmd: &str, args: &str) -> Result<std::process::ExitStatus, InternalEr
     Ok(status)
 }
 
+fn check_if_the_module_with_version_installed(
```

Comment:
I would love to see a new unit-test for that function. Is it feasible or to hard to mock?

Reply:
I feel its harder to mock because this function involves the commands

Reply:
This function is useless and should be simply removed.